viewflow_extensions/views.py

```diff
@@ -61,3 +60,17 @@ def get_success_url(self):
         if self._save:
             return self.request.get_full_path()
         return super().get_success_url()
+
+
+def make_form_class_without_required_fields(form_class):
+
+    class NoRequiredFieldsForm(form_class):
```

Comment:
hm... I know this works, but I don't like it. This could have unwanted side effects.
I would prefer to just modify the form instance, rather than creating a new class. This might have unexpected side effects and makes debugging more difficult.